diff --git a/TestCases/compliance-level-3/1120-feel-duration-function/1120-feel-duration-function.dmn b/TestCases/compliance-level-3/1120-feel-duration-function/1120-feel-duration-function.dmn
index fce6474..2d7627b 100644
--- a/TestCases/compliance-level-3/1120-feel-duration-function/1120-feel-duration-function.dmn
+++ b/TestCases/compliance-level-3/1120-feel-duration-function/1120-feel-duration-function.dmn
@@ -1,8 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <definitions namespace="http://www.actico.com/spec/DMN/1120-feel-duration-function" name="1120-feel-duration-function" id="_jG2L8PUUEeesLuP4RHs4vA" xmlns="https://www.omg.org/spec/DMN/20211108/MODEL/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:triso="http://www.trisotech.com/2015/triso/modeling" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <description>FEEL built-in function 'duration(from [String])' in category conversion functions</description>
-    <itemDefinition name="tfeel-duration-function_001_f2c6cd6866_Result" id="_jG2L8vUUEeesLuP4RHs4vA"/>
-    <itemDefinition name="tfeel-duration-function_002_ddca5756ca_Result" id="_jG2L9vUUEeesLuP4RHs4vA"/>
+    <itemDefinition name="tfeel-duration-function_001_f2c6cd6866_Result" id="_jG2L8vUUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
+    <itemDefinition name="tfeel-duration-function_002_ddca5756ca_Result" id="_jG2L9vUUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
     <itemDefinition name="tfeel-duration-function_003_951e1d1c31_Result" id="_jG2zAPUUEeesLuP4RHs4vA">
         <typeRef>dayTimeDuration</typeRef>
     </itemDefinition>
@@ -117,16 +121,36 @@
     <itemDefinition name="tfeel-duration-function_040_7d8eae461f_Result" id="_jG7rifUUEeesLuP4RHs4vA">
         <typeRef>yearMonthDuration</typeRef>
     </itemDefinition>
-    <itemDefinition name="tfeel-duration-function_041_264bc9d682_Result" id="_jG7rjfUUEeesLuP4RHs4vA"/>
-    <itemDefinition name="tfeel-duration-function_042_59a0000245_Result" id="_jG7rkfUUEeesLuP4RHs4vA"/>
-    <itemDefinition name="tfeel-duration-function_043_253815dc6c_Result" id="_jG7rlfUUEeesLuP4RHs4vA"/>
-    <itemDefinition name="tfeel-duration-function_044_f3b338d877_Result" id="_jG8Sk_UUEeesLuP4RHs4vA"/>
-    <itemDefinition name="tfeel-duration-function_045_2ffcc37801_Result" id="_jG8Sl_UUEeesLuP4RHs4vA"/>
-    <itemDefinition name="tfeel-duration-function_046_eb637de5f6_Result" id="_jG8Sm_UUEeesLuP4RHs4vA"/>
-    <itemDefinition name="tfeel-duration-function_047_3210c46a5a_Result" id="_jG8Sn_UUEeesLuP4RHs4vA"/>
-    <itemDefinition name="tfeel-duration-function_048_ab6244f767_Result" id="_jG8So_UUEeesLuP4RHs4vA"/>
-    <itemDefinition name="tfeel-duration-function_049_2225b503a0_Result" id="_jG85o_UUEeesLuP4RHs4vA"/>
-    <itemDefinition name="tfeel-duration-function_050_dd2ef33bbd_Result" id="_jG85p_UUEeesLuP4RHs4vA"/>
+    <itemDefinition name="tfeel-duration-function_041_264bc9d682_Result" id="_jG7rjfUUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
+    <itemDefinition name="tfeel-duration-function_042_59a0000245_Result" id="_jG7rkfUUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
+    <itemDefinition name="tfeel-duration-function_043_253815dc6c_Result" id="_jG7rlfUUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
+    <itemDefinition name="tfeel-duration-function_044_f3b338d877_Result" id="_jG8Sk_UUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
+    <itemDefinition name="tfeel-duration-function_045_2ffcc37801_Result" id="_jG8Sl_UUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
+    <itemDefinition name="tfeel-duration-function_046_eb637de5f6_Result" id="_jG8Sm_UUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
+    <itemDefinition name="tfeel-duration-function_047_3210c46a5a_Result" id="_jG8Sn_UUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
+    <itemDefinition name="tfeel-duration-function_048_ab6244f767_Result" id="_jG8So_UUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
+    <itemDefinition name="tfeel-duration-function_049_2225b503a0_Result" id="_jG85o_UUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
+    <itemDefinition name="tfeel-duration-function_050_dd2ef33bbd_Result" id="_jG85p_UUEeesLuP4RHs4vA">
+        <typeRef>yearMonthDuration</typeRef>
+    </itemDefinition>
     <decision name="feel-duration-function_ErrorCase_001_f2c6cd6866" id="_jG2L9PUUEeesLuP4RHs4vA">
         <description>Tests FEEL expression: 'duration(null)' and expects result: 'null (null)'</description>
         <question>Result of FEEL expression 'duration(null)'?</question>
