diff --git a/TestCases/compliance-level-3/1146-feel-context-put-function/1146-feel-context-put-function.dmn b/TestCases/compliance-level-3/1146-feel-context-put-function/1146-feel-context-put-function.dmn
index 73fc3fd..19912af 100644
--- a/TestCases/compliance-level-3/1146-feel-context-put-function/1146-feel-context-put-function.dmn
+++ b/TestCases/compliance-level-3/1146-feel-context-put-function/1146-feel-context-put-function.dmn
@@ -236,7 +236,7 @@
     <decision name="nested009" id="_nested009">
         <variable name="nested009"/>
         <literalExpression>
-            <text>context put({x:1, y: {a: 0}}, ["y", "a", "b", "c"], 2)</text>
+            <text>context put(context: {x:1, y: {a: 0}}, ky: ["y", "a", "b", "c"], value: 2)</text>
         </literalExpression>
     </decision>
 
